client/src/main/proto/spine/event_annotations.proto

```diff
@@ -51,6 +51,7 @@ extend google.protobuf.FieldOptions {
     // </pre>
     //
     // The enrichment message can be defined anywhere and with any name.
+    // The value of the option is the name of the source event field, exactly the same as in the definition.
```

Comment:
As we discussed over Skype, please implement the definition of source field references using type name and FQN, if an enrichment is in another package, and revert the documentation of such abilities here.